.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/sdl/SDL_image"]
 	path = lib/sdl/SDL2_image
 	url = https://github.com/SDL-mirror/SDL_image.git
+[submodule "lib/sdl/SDL2_gfx"]
+	path = lib/sdl/SDL2_gfx
+	url = https://github.com/Teufelchen1/libSDL_gfx.git
```

Comment:
Not sure if necessary here, but instead of reuploading the files without history, you can use "Import Repository" for `https://svn.code.sf.net/p/sdl2gfx/code/trunk` on GitHub to end up with the full SVN history.

Unfortunately this still won't allow to keep the repository synchronized with the SVN counterpart.
There are other solutions and tools to do that though (if necessary).

However, frustratingly, the author did not use tags and there's no commit from the official release date of the source package either. So it looks like they released a version that's not checked or marked in their source-control.
So it's questionable if there's any benefit from the history.